backend/db_init.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE IF NOT EXISTS users(
```

Comment:
can we remove the user table? we should just use the `user_id` field from Auth0 to reference other objects I think.